.cci.jenkinsfile

```diff
@@ -5,7 +5,8 @@ parallel rpms: {
   def n = 5
   cosaPod(buildroot: true, runAsUser: 0, memory: "2Gi", cpu: "${n}") {
       checkout scm
-      shwrap("""RPM_BUILD_NCPUS=${n} ./ci/coreosci-rpmbuild.sh
+      // 2:1 job to CPU at most should keep us from getting kicked out
+      shwrap("""RPM_BUILD_NCPUS=${n} CARGO_BUILD_JOBS=${n} ./ci/coreosci-rpmbuild.sh
```

Comment:
Hmm shouldn't cargo respect the make jobserver?  https://github.com/rust-lang/cargo/pull/4110

Reply:
(Eh we can look at this later)

Reply:
Hmm, seems like it would.

Ohhh, I think it might be the cmake call for libdnf which is not capped. I wonder if the issue is that the jobserver isn't surviving the `make` -> `cmake` -> `make` chain. Briefly searching around, it's not clear to me whether cmake supports using a parent job server for the children make processes.

Next time we hit the libgomp thing, let's try something like this instead:

```diff
diff --git a/.cci.jenkinsfile b/.cci.jenkinsfile
index 5cd9e699..29d11897 100644
--- a/.cci.jenkinsfile
+++ b/.cci.jenkinsfile
@@ -6,7 +6,7 @@ parallel rpms: {
   cosaPod(buildroot: true, runAsUser: 0, memory: "2Gi", cpu: "${n}") {
       checkout scm
       // 2:1 job to CPU at most should keep us from getting kicked out
-      shwrap("""RPM_BUILD_NCPUS=${n} CARGO_BUILD_JOBS=${n} ./ci/coreosci-rpmbuild.sh
+      shwrap("""RPM_BUILD_NCPUS=${n} CMAKE_BUILD_PARALLEL_LEVEL=${n} ./ci/coreosci-rpmbuild.sh
                 mv packaging/*/*.rpm .
              """)
       // make it easy for anyone to download the RPMs
```

Hmm, or maybe that should be in the spec file instead since `RPM_BUILD_NCPUS` really should be the only top-level knob.

Reply:
> Ohhh, I think it might be the cmake call for libdnf which is not capped. I wonder if the issue is that the jobserver isn't surviving the make -> cmake -> make chain.

I think it should all be inherited by the environment by default?  Unless the child make explicitly invokes `make -jN` which AIUI resets things.